antidox/perspective_test.py

```diff
@@ -185,7 +184,50 @@ def test_contains_toxicity_false(self):
+    I believe it was the former Franklin Shuttle Station next door. Well, since both are officially in Crown Heights, we really need better names for both of them now.    
+     I shan’t claim to know; sometimes I’m not even sure whether I’ve crossed into Queens.   
+     Don't you see the "You are now leaving Brooklyn - Fugeddabouddit" signs?   
+     In his defense, those signs can't be at every street that crosses the Brooklyn-Queens line. As far as the Studebaker Buildings, since both are officially considered to be in Crown Heights, how about renaming the gallery for the landmark "Studebaker Building (Bedford Avenue, Brooklyn)," and I'll rename the other one "Studebaker Building (Dean Street, Brooklyn)?" I'll even correct Dean Street (BMT Franklin Avenue Line).-   
+
+    Buzz Aldrin in project scope?
+    Why would Buzz Aldrin be in  scope?    
+    No idea. I've removed it.   
+     WP 1.0 Bot Beta 
+    Error: <HttpError 400 when requesting https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=AIzaSyA5kva-B5eLUBMD-VB426Pan3AlPXzWOXc&alt=json returned "Attribute THREAT does not support request languages: hu">
+
+    Hello! Your WikiProject has been selected to participate in the WP 1.0 Bot rewrite beta. This means that, starting in the next few days or weeks, your assessment tables will be updated using code in the new bot, codenamed Lucky. You can read more about this change on the Wikipedia 1.0 Editorial team page. Thanks!   
+
+     A new newsletter directory is out! 
+
+    A new '''Newsletter directory''' has been created to replace the old, out-of-date one. If your WikiProject and its taskforces have newsletters (even inactive ones), or if you know of a missing newsletter (including from sister projects like WikiSpecies), please include it in the directory! The template can be a bit tricky, so if you need help, just post the newsletter on the template's talk page and someone will add it for you.
+    – Sent on behalf of . 
+
+     Help with a park article? 
+
+    Hi! I didn't know if anyone was willing to work on a park article or not - a student of mine created the article on St. James Park (Bronx). The class ends this week and I'm not entirely sure if they will be back on to edit it, but at the present it lacks information and sourcing to establish how it's notable. I'm going to try to do as much as I can for it, but I'm admittedly kind of swamped with other classes so I wanted to see if anyone would be interested in this. 
+
+     Wikipedia:Naming conventions (US stations)/NYC Subway RfC 
+
+    Just so everyone who would come here knows, there is an ongoing RfC at Wikipedia:Naming conventions (US stations)/NYC Subway RfC that WP:NYC might be interested in.       @  """
+    text = perspective.get_wikipage("Wikipedia_talk:WikiProject_New_York_City")
+    self.assertTrue(text, clean_text)
```

Comment:
This test currently depends on what Wikipedia returns, so if anyone adds or makes changes to the "Wikipedia_talk:WikiProject_New_York_City" page, your test may no longer pass.

I think it would be better to modify to add a new function that just takes the Wikipedia response text (what you currently have as `get_page.content` in perspective.py) and returns a cleaned string of the content, minus any markup.

Also it would be good to test with a smaller string that's more readable, maybe you can find another talk page that has just a few comments, but not as many as this NYC page.

Reply:
I don't think I understand this comment.